temporal_sqlalchemy/clock.py

```diff
@@ -221,6 +195,46 @@ def build_clock_class(
     return type('%sClock' % name, base_classes, props)
 
 
+def build_clock_table(entity_table: sa.Table,
+                      metadata: sa.MetaData,
+                      schema: str,
+                      activity_class=None) -> sa.Table:
+    clock_table_name = util.truncate_identifier(
+        "%s_clock" % entity_table.name)
+    clock_table = sa.Table(
+        clock_table_name,
+        metadata,
+        sa.Column('tick',
```

Comment:
@multigl 
Could we approve and release for PR #31 for issue #26 first before this refactor?

Or do you want all schema changes to be in one release?